docs/ru/guides/testing-async-components.md

```diff
@@ -44,7 +44,9 @@ export default {
 ```js
 import { shallowMount } from '@vue/test-utils'
 import Foo from './Foo'
-jest.mock('axios')
+jest.mock('axios', () => ({
+  get: Promise.resolve('value')
```

Comment:
Since the example does `this.value = response.data`, should this be

```js
jest.mock('axios', () => ({
  get: Promise.resolve({ data: 'value' })
}))
```